flecs_ecs/examples/flecs/queries/query_group_by_callbacks.rs

```diff
@@ -32,8 +32,8 @@ pub struct Third;
 #[derive(Component)]
 pub struct Group;
 
-// callbacks need to be extern "C" to be callable from C
-extern "C" fn callback_group_create(
+// callbacks need to be extern "C-unwind" to be callable from C
```

Comment:
This comment (repeated below) should be updated a bit since `"C-unwind"` has the improved semantics, not just "callable from C".

Reply:
one option would be
> callbacks need to be extern "C-unwind"

since, well, (new) flecs-rust API demands it 🤷 

---

alternatively something like this?
> callbacks need to be extern "C-unwind" to be callable from C (with unwinding support)

Reply:
updated it to `Callbacks need to be `extern "C-unwind"` to be callable from C and allow safe unwinding across FFI boundaries.`